setup.py

```diff
@@ -47,6 +47,10 @@
     entry_points={'console_scripts': [
         'django-admin = django.core.management:execute_from_command_line',
     ]},
+    install_requires=[
+        'asgiref>=0.10',
+        'daphne>=0.11',
```

Comment:
`asgiref` has extensive test coverage on the parts we are using (in fact, one of its main jobs is to provide the test confirmance suite). Daphne is less so - see my post to django-developers.